tests/transformers/test_numerical.py

```diff
@@ -33,6 +33,194 @@ def test_fit(self):
+        # Run
+        transformer = NumericalTransformer(dtype=np.float, nan='nan', rounding='auto')
+        transformer.fit(data)
+
+        # Asserts
+        assert transformer.rounding_digits is None
+
+    def test_reverse_transform_rounding_none(self):
+        """Test ``reverse_transform`` when ``rounding`` is ``None``
+
+        The data should not be rounded at all.
+        """
+        # Setup
+        data = np.random.random(10)
+
+        # Run
+        transformer = NumericalTransformer(dtype=np.float, nan=None)
+        transformer.rounding_digits = None
+        result = transformer.reverse_transform(data)
+
+        # Assert
+        assert np.array_equal(result, data)
+
+    def test_reverse_transform_rounding_positive_int(self):
+        """Test ``reverse_transform`` when ``rounding`` is a positive int
+
+        The data should round to the maximum number of decimal places
+        set in the ``rounding_digits`` value.
+
+        Input:
+        - Array with decimals
+
+        Output:
+        - Same array rounded to the provided number of decimal places
+        """
+        # Setup
+        data = np.array([1.1111, 2.2222, 3.3333, 4.44444, 5.555555])
+
+        # Run
+        transformer = NumericalTransformer(dtype=np.float, nan=None)
+        transformer.rounding_digits = 2
+        result = transformer.reverse_transform(data)
+
+        # Assert
+        expected_data = np.array([1.11, 2.22, 3.33, 4.44, 5.56])
+        assert np.array_equal(result, expected_data)
+
+    def test_reverse_transform_rounding_negative_int(self):
+        """Test ``reverse_transform`` when ``rounding`` is a negative int
+
+        The data should round to the number set in the ``rounding_digits``
+        attribute.
+
+        Input:
+        - Array with with larger numbers
+
+        Output:
+        - Same array rounded to the provided number of 0s
+        """
+        # Setup
+        data = np.array([2000, 120, 3100, 40100])
+
+        # Run
+        transformer = NumericalTransformer(dtype=np.float, nan=None)
+        transformer.rounding_digits = -3
+        result = transformer.reverse_transform(data)
+
+        # Assert
+        expected_data = np.array([2000, 0, 3000, 40000])
+        assert np.array_equal(result, expected_data)
+
+    def test_reverse_transform_rounding_zero(self):
+        """Test ``reverse_transform`` when ``rounding`` is a negative int
+
+        The data should round to the number set in the ``rounding_digits``
+        attribute.
+
+        Input:
+        - Array with with larger numbers
+
+        Output:
+        - Same array rounded to the provided number of 0s
+        """
+        # Setup
+        data = np.array([2000.554, 120.2, 3101, 4010])
+
+        # Run
+        transformer = NumericalTransformer(dtype=np.float, nan=None)
+        transformer.rounding_digits = 0
+        result = transformer.reverse_transform(data)
+
+        # Assert
+        expected_data = np.array([2001, 120, 3101, 4010])
```

Comment:
Should this not be a `float` array?

Reply:
I suppose. I did this because `[2001, 120, 3101, 4010]` and `[2001.0, 120.0, 3101.0, 4010.0]` will pass as equivalent